src/PrestaShopBundle/Form/Admin/Sell/Catalog/AttributeGroupType.php

```diff
@@ -85,7 +85,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                         ]),
                     ],
                 ],
-                'help' => $this->trans('Your internal name for this attribute.', 'Admin.Catalog.Help')
+                'help' => $this->trans('The public name for this attribute, displayed to the customers.', 'Admin.Catalog.Help')
```

Comment:
You probably need to add constraints in this form type (and maybe in the `AttributeType` as well?)

Reply:
There is already a constraint for the public name here.

Reply:
It would be maybe better if this field was optional and was autofilled with the normal name.

Name and public name are the same in most cases, would save some typing...

ping @MatShir @kpodemski